pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -341,24 +387,20 @@ func (b *OFBridge) AddFlowsInBundle(addflows []Flow, modFlows []Flow, delFlows [
 		return nil
 	}
 
-	// Install new Openflow entries with the opened bundle.
-	if err := syncFlows(addflows, openflow13.FC_ADD); err != nil {
-		return err
-	}
-	// Modify existing Openflow entries with the opened bundle.
-	if err := syncFlows(modFlows, openflow13.FC_MODIFY_STRICT); err != nil {
-		return err
-	}
-	// Delete Openflow entries with the opened bundle.
-	if err := syncFlows(delFlows, openflow13.FC_DELETE_STRICT); err != nil {
-		return err
+	// Add Group modification messages in advance of Flow modification messages.
```

Comment:
Could you explain why this is required?

Reply:
Sure, will add comments. The reason is Flow entries might have dependency on the Group entry. I have changed the Bundle to add message in order, and this order could ensure the Group is existing when adding Flow entry. As for the deletion case, the corresponding Flow entry is removed when deleting the Group entry, and no error returns when OVS is trying to delete a non-existing Flow.